### ISSUE TITLE
[SignalR] Problem with grid reloading

### ISSUE THREAD

**Issue by DonjetSalihi** [NonFactors/AspNetCore.Grid#308]
So what i am doing is i am using signal R when i am clicking a button i am trying to add a new row in page real time. So the code of adding is this: 
`  private async Task StartGetOrders()
        {
            try
            {
                for (int i = 0; i < 10; i++)
                {
                    await Task.Delay(1000);
                    Person person = new Person();
                    person.Name = "a";
                    person.Surname = "a";
                    person.MaritalStatus = "a";
                    person.Age = "a";
                    person.Birthday = "a";
                    person.IsWorking = "a";
                    peopleList.Add(person);
                    People = peopleList.AsQueryable();
                    await _progressHubContext.Clients.Client(ConnectionId).SendAsync("initProgressBar");
                }
            }
            catch(Exception ex)
            {
            }
        }`

and the jquery is this:
 ` progressConnection.on("initProgressBar", () => {
                var grid = new MvcGrid($('.mvc-grid')[0]);
                grid.reload();
            });`  
Basically it finds the mvc grid in the jquery code but it doesn't update the grid real time.
Can you please tell me what i am doing wrong ? Thanks in advance.

**Comment by Muchiachio** [comment]
Doesn't seem anything wrong. You can the debug grid's data endpoint to see if it's hit and it returns the new data.

**Comment by DonjetSalihi** [comment]
Hi Muchiachio, i managed to make it work by using the AjaxGrid and making some code changes. Thanks for the fast reply.
